mne/io/edf/edf.py

```diff
@@ -495,9 +473,10 @@ def _get_edf_info(fname, stim_channel, annot, annotmap, eog, misc, exclude,
         info['highpass'] = float(np.max(highpass))
         warn('Channels contain different highpass filters. Highest filter '
              'setting will be stored.')
-
     if lowpass.size == 0:
-        pass
+        info['sfreq'] / 2.
```

Comment:
just realised that `_empty_info(sfreq)` defaults to `lowpass = sfreq/2`, so I won't make any assignment here.